impeller/entity/entity_pass.cc

```diff
@@ -252,7 +252,9 @@ EntityPass::EntityResult EntityPass::GetEntityForElement(
 
     auto subpass_coverage =
         GetSubpassCoverage(*subpass, Rect::MakeSize(root_pass_size));
-
+    if (subpass->cover_whole_screen) {
+      subpass_coverage = Rect::MakeSize(root_pass_size);
```

Comment:
Per my other comment, I believe making this just be the coverage of the parent pass would suffice.
```cpp
subpass_coverage = Rect(
  position, Size(pass_context.GetRenderTarget().GetRenderTargetSize()));
```

Reply:
Done.